src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -350,14 +350,45 @@ def switch_browser(self, index_or_alias):
         self.debug('Switched to browser with Selenium session id %s.'
                    % self.driver.session_id)
 
+    @keyword
+    def get_browser_ids(self):
+        """Returns index of all active browser as list.
+
+        See `Switch Browser` for more information and examples.
+
+        New in SeleniumLibrary 4.0
+        """
+        return self.drivers.active_driver_ids
+
+    @keyword
+    def get_browser_aliases(self):
+        """Returns aliases of all active browser that has an alias as NormalizedDict.
+        The dictionary contains the aliases as keys and the index as value.
+        This can be accessed as dictionary ``${aliases.key}`` or as list ``@{aliases}[0]``.
+
+        Example:
+        | `Open Browser` | https://example.com   | alias=BrowserA | |
+        | `Open Browser` | https://example.com   | alias=BrowserB | |
+        | &{aliases}     | `Get Browser Aliases` |                | # &{aliases} = { BrowserA=1|BrowserB=2 } |
+        | `Log`          | ${aliases.BrowserA}   |                | # logs ``1`` |
+        | FOR            | ${alias}              | IN             | @{aliases} |
+        |                | `Log`                 | ${alias}       | # logs ``BrowserA`` and ``BrowserB`` |
+        | END            |                       |                | |
+
+        See `Switch Browser` for more information and examples.
+
+        New in SeleniumLibrary 4.0
+        """
+        return self.drivers.active_aliases
+
     @keyword
     def get_session_id(self):
         """Returns the currently active browser session id.
 
         New in SeleniumLibrary 3.2
         """
         return self.driver.session_id
-
+    
```

Comment:
Nit pic, spaces in here are not allowed.

Reply:
I will fox this

Reply:
Done

Reply:
There are atests that are covering all of these Keyword!

I will do an example.